sdk/core/azure-core/azure/core/utils/_utils.py

```diff
@@ -161,3 +162,25 @@ def __eq__(self, other: Any) -> bool:
 
     def __repr__(self) -> str:
         return str(dict(self.items()))
+
+
+def parse_status_link(status_link: str, lro_options: Optional[Dict[str, Any]] = None) -> Tuple[str, Dict[str, Any]]:
+    """Handle status link.
+
+    :param status_link: Lro status link.
+    :type status_link: str
+    :param lro_options: Options for LRO.
+    :type lro_options: dict[str, Any]
+    :return: Parsed status link and parsed query parameters.
+    :rtype: Tuple[str, Dict[str, Any]]
+    """
+    request_params: Dict[str, Any] = {}
+    if lro_options and "api_version" in lro_options:
```

Comment:
Why we ignore lro_options and only return {} as request_params if "api_version" not in lro_options?

Reply:
We only want to add or replace api_vesrion of status link if lro_options configure it. Otherwise we do nothing. It shall be much easier to understand with context https://github.com/Azure/azure-sdk-for-python/issues/33263